language/movie_nl/README.md

```diff
@@ -0,0 +1,146 @@
+# Introduction
+This sample is an application of the Google Cloud Platform Natural Language API.
+It uses the [imdb movie reviews data set](https://www.cs.cornell.edu/people/pabo/movie-review-data/) 
+from [Cornell University](http://www.cs.cornell.edu/) and performs sentiment & entity
+analysis on it. It combines the capabilities of sentiment analysis and entity recognition
+to come up with actors/directors who are the most and least popular.
+
+### Set Up to Authenticate With Your Project's Credentials
+
+Please follow the [Set Up Your Project](https://cloud.google.com/natural-language/docs/getting-started#set_up_your_project)
+steps in the Quickstart doc to create a project and enable the
+Cloud Natural Language API. Following those steps, make sure that you
+[Set Up a Service Account](https://cloud.google.com/natural-language/docs/common/auth#set_up_a_service_account),
+and export the following environment variable:
+
+```
+export GOOGLE_APPLICATION_CREDENTIALS=/path/to/your-project-credentials.json
```

Comment:
I would like to seea  big fat warning about an issue that monitoring also had. If you screw this step up (and environment variables are one of the top things people screw up) you get a misleading error message that the API still isn't enabled and you should wait a few minutes. We need to warn people that they might also need to double check this is set correctly.